src/pages/cli/graphql/search-and-result-aggregations.mdx

```diff
@@ -0,0 +1,378 @@
+export const meta = {
+  title: `Search and result aggregations`,
+  description: `Add authorization rules to your GraphQL schema to control access to your data.`,
+};
+
+Add the `@searchable` directive to an `@model` type to enable OpenSearch-based data search and result aggregations. This gives you the ability to:
+- search for data using advanced filters, such as substring matching, wildcards, regex, `and`/`or`/`not` conditions
+- get aggregation values, such as total counts, sum, average, min, max
```

Comment:
total counts is not part of aggregation. Currently we support SUM, AVG, MIN, MAX and TERMS only.
`get aggregation values, such as sum, average, min, max`

Reply:
For the customer, it's an "aggregation" functionality, even though it might be exposed under a slightly different API.